src/pages/landelijk/gehandicaptenzorg.tsx

```diff
@@ -0,0 +1,217 @@
+import { useRouter } from 'next/router';
+import CoronaVirus from '~/assets/coronavirus.svg';
+import Locatie from '~/assets/locaties.svg';
+import Verpleeghuiszorg from '~/assets/verpleeghuiszorg.svg';
+import { ChoroplethTile } from '~/components-styled/choropleth-tile';
+import { ContentHeader } from '~/components-styled/content-header';
+import { KpiTile } from '~/components-styled/kpi-tile';
+import { KpiValue } from '~/components-styled/kpi-value';
+import { LineChartTile } from '~/components-styled/line-chart-tile';
+import { TwoKpiSection } from '~/components-styled/two-kpi-section';
+import { Text } from '~/components-styled/typography';
+import { regionThresholds } from '~/components/choropleth/region-thresholds';
+import { SafetyRegionChoropleth } from '~/components/choropleth/safety-region-choropleth';
+import { createSelectRegionHandler } from '~/components/choropleth/select-handlers/create-select-region-handler';
+import { createDisablityInfectedLocationsRegionalTooltip } from '~/components/choropleth/tooltips/region/create-disability-infected-locations-regional-tooltip';
+import { FCWithLayout } from '~/components/layout';
+import { getNationalLayout } from '~/components/layout/NationalLayout';
+import { SEOHead } from '~/components/seoHead';
+import siteText from '~/locale/index';
+import {
+  getNationalStaticProps,
+  NationalPageProps,
+} from '~/static-props/nl-data';
+
+const infectedLocationsText = siteText.gehandicaptenzorg_besmette_locaties;
+const positiveTestedPeopleText =
+  siteText.gehandicaptenzorg_positief_geteste_personen;
+const locationDeaths = siteText.gehandicaptenzorg_oversterfte;
+
+const DisabilityCare: FCWithLayout<NationalPageProps> = (props) => {
+  const { data } = props;
+  const disabilityCareData = data.disability_care;
+
+  const router = useRouter();
+
+  return (
+    <>
+      <SEOHead
+        title={infectedLocationsText.metadata.title}
+        description={infectedLocationsText.metadata.description}
+      />
+
+      <ContentHeader
+        category={siteText.nationaal_layout.headings.kwetsbare_groepen}
+        screenreaderCategory={
```

Comment:
Maybe it didn't originate from this PR, but a screen reader is not a single word.

```suggestion
        screenReaderCategory={
```